config.json.example

```diff
@@ -74,5 +74,9 @@
         "APP_ID": "appId",
         "KEY": "key",
         "SECRET": "secret"
+    },
+    "SLACK": {
+        "FLAGGING_URL": "https://hooks.slack.com/services/id/id/id",
+        "FLAGGING_FOOTER_LINK": "https://crookedneighbor.github.io/flag-o-rama/"
```

Comment:
This is a small webpage I created to make flagging easier. 

The slack message will contain a link to this page with query params for the group id and the chat id. If the mod has entered their api details (creds saved in localstorage), it will automatically flag them. Now that it's possible to flag and unflag messages in private groups, this will make it easier to flag inappropriate content that has been flagged once in a private group.

If we decide to merge this in, I'll copy this repo over to the HabitRPG org.